src/components/Menu/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import { socialLinks } from './utils/socialLinks'
+
+interface DropdownProps {
+  children: React.ReactNode
+}
+
+export function Dropdown({
+  children
+}: DropdownProps) {
```

Comment:
I think this could be more maintainable with the following points:

1. `socialLinks` could be a variable inside this file, don't think we need to split it into another file, it's not a big deal to keep it here;
3. I don't like receiving only the `children` property here being the trigger to the dropdown, I think we are losing all of the composability of Radix as we are not allowing the parent component to change the behavior of the dropdown somehow so, if we gonna keep all the dropdown behavior inside this component we could name this component `SocialLinksMenuButtton` and we can use MenuButton directly inside the `DropdownMenu.Trigger` instead of passing it as a property;

Reply:
done, I agree with the points. When I started coding, my doubt was, will this component be 'global' or to fulfill only to social links.